### ISSUE TITLE
[Bug] UI Bug

### ISSUE THREAD

**Issue by RoyBatty64** [TES5Edit/TES5Edit#397]
Holding shift and double clicking a field to edit it like script source or a message sometimes screws up desktop rendering. The screen will flash as it opens the edit window and other windows will go black and have to be closed and re-opened. Refreshing with F5 does not work.

OS Windows 7 x64

**Comment by zilav** [comment]
Never had such issue. If you have steps to reproduce, then I can take a look.

**Comment by RoyBatty64** [comment]
How to reproduce, holding shift and clicking a field to edit, intermittent behavior try doing it 10-20 times and it should happen. It seems to happen most often with script source, but also happens in notes and messages although less frequently. I checked my event log and there is this error "The Desktop Window Manager has encountered a fatal error (0x88980406)". This only happens with xEdit an no other program.

I think it's possibly an issue with multiple monitors. I noticed if you size the window larger than 1920 width it will reset the window size on restart to whatever the default is. It also does this if I close it while it's on my secondary monitor.
